plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetReaderConfig.java

```diff
@@ -98,6 +98,19 @@ public boolean isUseColumnIndex()
         return options.isUseColumnIndex();
     }
 
+    @Config("parquet.use-bloom-filter")
+    @ConfigDescription("Enable using Parquet bloom filter")
+    public ParquetReaderConfig setUseBloomFilter(boolean useBloomFilter)
+    {
+        options = options.withUseBloomFilterIndex(useBloomFilter);
+        return this;
+    }
+
+    public boolean isUseBloomFilter()
```

Comment:
`useBloomFilter`

Reply:
hi, I tried to rename from isUseBloomFilter() to useBloomFilter()

This result in unit test TestParquetReaderConfig failed. 
with error:
`No getter for @Config method [public ParquetReaderConfig ParquetReaderConfig.setUseBloomFilter(boolean)].`

with stacktrace:

`
	at io.airlift.configuration.Problems.getException(Problems.java:135)
	at io.airlift.configuration.Problems.throwIfHasErrors(Problems.java:73)
	at io.airlift.configuration.ConfigurationMetadata.getValidConfigurationMetadata(ConfigurationMetadata.java:53)
	at io.airlift.configuration.ConfigurationMetadata.getValidConfigurationMetadata(ConfigurationMetadata.java:46)
	at io.airlift.configuration.testing.ConfigAssertions.assertRecordedDefaults(ConfigAssertions.java:244)
	at io.trino.plugin.hive.parquet.TestParquetReaderConfig.testDefaults(TestParquetReaderConfig.java:33)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
	at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)`

Reply:
I think we can rename the property to `parquet.bloom-filters.enabled` instead and avoid the awkward `isUse...` naming.

Reply:
hi, would like to reserve "parquet.bloom-filters.enabled" for specifying writing parquet: similar to https://github.com/apache/parquet-mr/blob/master/parquet-hadoop/README.md's  parquet.bloom.filter.enabled.

parquet.use-bloom-filter means using bloomfilters when reading, similar to parquet.use-column-index